app/api/v1/endpoints/events.py

```
@@ -0,0 +1,29 @@
+from log.logger import get_app_logger
+from fastapi import Depends, HTTPException
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+
+
+logger = get_app_logger(__name__)
+
+router = APIRouter()
+security = HTTPBearer()
+
+def verify_bearer_token(
+    credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)